cpp/include/raft/spatial/knn/knn.cuh

```diff
@@ -224,4 +345,84 @@ void brute_force_knn(raft::handle_t const& handle,
                                metric,
                                metric_arg);
 }
+
+/**
+ * @brief Flat C++ API function to perform a brute force knn on
+ * a series of input arrays and combine the results into a single
+ * output array for indexes and distances. Inputs can be either
+ * row- or column-major but the output matrices will always be in
+ * row-major format.
+ *
+ * @example
+ *
+ *
+ *
+ * @param[in] handle the cuml handle to use
+ * @param[in] index vector of device matrices (each size m_i*d) to be used as the knn index
+ * @param[in] search matrix (size n*d) to be used for searching the index
+ * @param[out] indices matrix (size n*k) to store output knn indices
+ * @param[out] distances matrix (size n*k) to store the output knn distance
+ * @param[in] k the number of nearest neighbors to return
+ * @param[in] metric_arg the value of `p` for Minkowski (l-p) distances. This
```

Comment:
It looks like `metric_arg` comes after `metric` in the parameter list, but before in the documentation.